### DISCUSSION TITLE
Change root directory (for file picker)

### DISCUSSION THREAD

**Discussion by thomasaarholt** [helix-editor/helix#8058] [Q&A] [answered]
Hello! I work in a repo that contains many projects, and when I open the file picker, I only want to list files within that project. My workflow could be something like this:
```bash
cd repo
cd projects
cd myproject
hx .
f
```
Unfortunately for me, the file picker looks for .git folders, and there is only one in the `repo` directory. I see that there exists the `:change-current-directory` command, but that doesn't affect the file picker, and there is no `:change-root-directory` command.

Does anyone have any advice? I'm aware that I can create a .git folder in my project directory, but that is rather confusing behaviour.

**Comment by CptPotato** [comment]
Do you mean you want only files from the current directory (`:pwd` in helix)? That works using <kbd>Space</kbd><kbd>F</kbd> (uppercase F)

You can also `:cd` in helix if you want to change the current directory.

**Reply by thomasaarholt** [reply, marked as answer]
Ah! The first point is what I want :) Thanks!

**Comment by homebrewmellow** [comment]
it is so jarring if you press <space>f and are bombarded by 40000 files. and the top results are porn in your download folder 🤣
